src/Sidekick.Modules.Trade/Components/Filters/FiltersComponent.razor

```diff
@@ -100,9 +99,7 @@
             </div>
         }
 
-        <div class="px-1 flex">
```

Comment:
This is necessary, otherwise there's a scrollbar shown in the popover for this item.